init.zsh

```diff
@@ -2,14 +2,14 @@
 # Zim initializition
 #
 
-autoload -Uz is-at-least
-if ! is-at-least 5.2; then
-  print "ERROR: Zim didn't start. You're using zsh version ${ZSH_VERSION}, and versions < 5.2 are not supported. Update your zsh." >&2
-  return 1
-fi
+# autoload -Uz is-at-least
+# if ! is-at-least 5.2; then
+#   print "ERROR: Zim didn't start. You're using zsh version ${ZSH_VERSION}, and versions < 5.2 are not supported. Update your zsh." >&2
+#   return 1
+# fi
 
 # Define zim location
-ZIM="${ZDOTDIR:-${HOME}}/.zim"
+(( ! ${+ZIM_HOME} )) && ZIM_HOME="${ZDOTDIR:-${HOME}}/.zim"
```

Comment:
I see that @Eriner mentioned it before in https://github.com/Eriner/zim/issues/92#issuecomment-250933429 (pointing to this [spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html)), and that [our fasd module](https://github.com/Eriner/zim/blob/52b272ca3715250707be9bd6a5e61bcca1c8999c/modules/fasd/functions/fasd#L36) is trying to load xdg configs like this (which seems compliant to the specs):

    if [[ -s ${XDG_CONFIG_HOME:-"${HOME}/.config"}/fasd/config ]]; then
      source ${XDG_CONFIG_HOME:-"${HOME}/.config"}/fasd/config
    ...

Just not sure what's the right way to do it...